app/src/components/dialog/EditDialog.tsx

```diff
@@ -75,7 +75,7 @@ export const EditDialog: React.FC<IEditDialogProps> = (props) => {
             aria-labelledby="edit-dialog-title"
             aria-describedby="edit-dialog-description">
             <DialogTitle id="edit-dialog-title">{props.dialogTitle}</DialogTitle>
-            <DialogContent>{props?.component?.element}</DialogContent>
+            <DialogContent>{props.component.element}</DialogContent>
```

Comment:
we need to add the error state message here in this component and also add the tests for it I think as part of this PR